### ISSUE TITLE
Updates to focus states for existing components

### ISSUE THREAD

**Issue by quinnkeast** [sourcegraph/sourcegraph#19019]
### 🚨 This is a draft and not final

As part of our effort to improve accessibility and usability, we're looking at systemizing and improving how we handle focus states and keyboard navigation across Sourcegraph.

The basic focus state update is broadly applicable, as previewed on a few elements below:

**Buttons, toggles, filter cards, etc.:**

<img width="658" alt="CleanShot 2021-03-11 at 13 00 17@2x" src="https://user-images.githubusercontent.com/6304497/110784252-c3051800-8269-11eb-9db2-f0a723eb5715.png">

**Text inputs and selects:**

<img width="822" alt="CleanShot 2021-03-11 at 13 01 01@2x" src="https://user-images.githubusercontent.com/6304497/110784328-d912d880-8269-11eb-985d-a35499aed01f.png">

[Full writeup and design specs can be found in Figma](https://www.figma.com/file/2hmoakevpROtdxAZbR7Y8s/Application-and-Navigation-Areas-and-Behaviour?node-id=741%3A2297).

For the first actionable steps, we need to update focus states for the following components:

**Buttons, toggles, filter cards, etc.:**

- [ ] Primary button
- [ ] Secondary button
- [ ] Success button
- [ ] Info button
- [ ] Merged button
- [ ] Warning button
- [ ] Danger button
- [ ] Text button
- [ ] Dropdown button
- [ ] Button groups
- [ ] Filter chip
- [ ] Context selector
- [ ] Search toggles
- [ ] On/off toggle
- [ ] Filter line item
- [ ] Cards (in code monitor lists)
- [ ] Pagination buttons

**Text inputs and selects:**

- [ ] Text input
- [ ] Select

Size variants should be included.

Once we have implemented and finalized these focus states, we will add them to the component library in Figma.

**Comment by pdubroy** [comment]
@umpox I believe a good chunk of this was handled by the [global form field updates](https://github.com/sourcegraph/sourcegraph/pull/20302) that we did as part of the redesign, is that right? If so, can you update this issue to describe what the remaining work is (if any)?

cc @valerybugakov too as I believe you worked on some of these as well

**Comment by pdubroy** [comment]
I'm going to close this issue, under the assumption that (a) the focus states are already implemented, or (b) they are covered by issues for the individual Wildcard components (see #27606). So I don't think this issue on its own is useful to us anymore.
